controllers/administration.js

```diff
@@ -973,6 +973,24 @@ router.post(
 	upload.single('csvFile'),
 	getCSVImportHandler(),
 );
+router.get(
+	'/teachers/import',
+	permissionsHelper.permissionsChecker(['ADMIN_VIEW', 'TEACHER_CREATE'], 'or'),
+	async (req, res, next) => {
+		const years = getSelectableYears(res.locals.currentSchoolData);
+		const title = `${returnAdminPrefix(
+			res.locals.currentUser.roles,
+		)}Schüler`;
+		res.render('administration/import', {
+			title,
+			roles: 'teacher',
+			action: `/administration/teacher/import?_csrf=${res.locals.csrfToken}`,
```

Comment:
ist hier auch die route /teachers?
```suggestion
			action: `/administration/teachers/import?_csrf=${res.locals.csrfToken}`,
```